usr/share/rear/verify/BAREOS/default/040_prepare_environment_vars.sh

```diff
@@ -0,0 +1,19 @@
+# read config vars defined during prep.
+source $VAR_DIR/bareos.conf
```

Comment:
Please make sure that we use this file only to carry over information that is not part of the configuration